spec/dummy/config/environments/development.rb

```diff
@@ -10,7 +10,7 @@
   config.eager_load = false
 
   # Show full error reports and disable caching.
-  config.consider_all_requests_local       = true
+  config.consider_all_requests_local       = false
```

Comment:
Set this to false if you want to see the behaviour in develop. I need to change it back to true in this PR